File: php/raindrops/raindrops.php
<?php

function raindrops($drops) {
    if (!is_numeric($drops)) {
        throw new \InvalidArgumentException();
    }

    $text = '';
    if ($drops % 3 == 0) {
        $text .= 'Pling';
    }
    if ($drops % 5 == 0) {
        $text .= 'Plang';
    }
    if ($drops % 7 === 0) {
        $text .= 'Plong';
    }
    if ($text === '') {
        $text = (string)$drops;
    }

    return $text;
}

File: go/raindrops/raindrops.go
package raindrops

import (
	"bytes"
	"fmt"
)

func Convert(drops int) string {
	var buffer bytes.Buffer

	if drops%3 == 0 {
		buffer.WriteString("Pling")
	}

	if drops%5 == 0 {
		buffer.WriteString("Plang")
	}

	if drops%7 == 0 {
		buffer.WriteString("Plong")
	}

	if buffer.Len() == 0 {
		return fmt.Sprint(drops)
	}

	return buffer.String()
}
